### ISSUE TITLE
p3GxsForums::getForumContent return all messages if unknown post id is passed

### ISSUE THREAD

**Issue by G10h4ck** [RetroShare/RetroShare#2388]
Passing an unknown post id to p3GxsForums::getForumContent I would expect no post is returned at all and eventually false as return value to indicate an error. Right now the behaviour is opposite, if the id of an unknown post is passed the all the posts are returned without reporting any error.

**Comment by gunjan543** [comment]
Hey @G10h4ck I wanted to contribute to "RetroShare Web Interface" in GSOC 2021, but I am not able to contact you, so please guide me on how should I proceed.

**Comment by G10h4ck** [comment]
> Hey @G10h4ck I wanted to contribute to "RetroShare Web Interface" in GSOC 2021, but I am not able to contact you, so please guide me on how should I proceed.

Sorry for late response, did you managed to contact with @csoler and to apply as student? I am not as active as other years in GSoC

**Comment by gunjan543** [comment]
Yes, I tried contacting him through email and element also, but I am not
able to connect with him.

**Comment by defnax** [comment]
@csoler @thunder2 close
